koji/collection-mash-split.py

```diff
@@ -385,7 +385,7 @@ def main():
             git_tag = "rpm/{}".format(version)
 
         CONFIG["comps_path"] = "/mnt/koji/mash/comps/foreman"
-        if collection == 'foreman':
+        if collection == 'foreman' and version in ['1.22', '1.23', '1.24', '2.0', '2.1']:
```

Comment:
```suggestion
        if collection == 'foreman' and version in ['1.24', '2.0', '2.1']:
```

Reply:
If you look at the file, there are references back to 1.22 in multiple places. I'd rather us clean that up all at once than just here in my PR.

Reply:
@ehelms please rebase, so 1f5eec57e490a25a77ecc95b34d01be31079b718 is included ;)